tests/integration/memcache/test_memcache_add.php82.php

```diff
@@ -0,0 +1,116 @@
+<?php
+/*
+ * Copyright 2020 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/*DESCRIPTION
+The agent should report Memcache metrics when keys are added using the Memcache
+procedural api.
+*/
+
+/*SKIPIF
+<?php require('skipif.inc');
+
+if (version_compare(PHP_VERSION, "8.2", "<")) {
+  die("skip: PHP 8.2 exclusive\n");
```

Comment:
More accurately isn't this "PHP 8.2+ exclusive"?  Maybe good to change the message or it could be misleading.

Reply:
Since the issue here is the memcache dependency itself, I think it may be better to keep it tied to 8.2, and if memcache releases the 8.2 fix that resolves this, this problem will be resolved for 8.3 and this can be removed entirely